dev/tasks/r/github.packages.yml

```diff
@@ -155,11 +186,19 @@ jobs:
           rtools-version: {{ '${{ matrix.r_version.rtools }}' }}
           Ncpus: 2
       {{ macros.github_setup_local_r_repo(false, true)|indent }}
+      - name: Install sccache
+        if: startsWith(matrix.platform, 'macos')
+        run: brew install sccache

```

Comment:
could we install on windows following the same approach we don on line 132: `- name: Get sccache`?
Maybe we could create our own action for that. I am just thinking out loud, not required for this PR.

Reply:
Yes I was also already thinking of making this it's own action.